api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirTaskServiceImpl.java

```diff
@@ -50,8 +50,8 @@
 	 */
 	@Override
 	@Transactional(readOnly = true)
-	public Task getTaskByUuid(String uuid) {
-		return translator.toFhirResource(dao.getTaskByUuid(uuid));
+	public Task getTask(String uuid) {
+		return translator.toFhirResource(dao.getTask(uuid));
```

Comment:
This shouldn't be necessary to declare here. I actually think all the methods except `searchForTask()` can be removed.